Update version to 2.51.0

dchakarwarti opened the pull request:
New ESPv2 release

dchakarwarti commented:
/retest

dchakarwarti commented:
/retest

TAOXUY commented:
Why do we need make a release? No any significant changes since last release? https://screenshot.googleplex.com/7W5hL98RKSgmm7w

dchakarwarti commented:
> Why do we need make a release? No any significant changes since last release? https://screenshot.googleplex.com/7W5hL98RKSgmm7w

There are some critical vulnerabilities observed in the `endpoints-runtime-serverless` image tracked in b/373177830.
https://screenshot.googleplex.com/8kF8c98DeWFdnkZ

TAOXUY commented:
> > Why do we need make a release? No any significant changes since last release? https://screenshot.googleplex.com/7W5hL98RKSgmm7w
> 
> There are some critical vulnerabilities observed in the `endpoints-runtime-serverless` image tracked in b/373177830. https://screenshot.googleplex.com/8kF8c98DeWFdnkZ

I see. What is the diff between the head and last release? So does the new docker build env fix all those vulnerabilities?https://github.com/GoogleCloudPlatform/esp-v2/pull/930/files

dchakarwarti commented:
> > > Why do we need make a release? No any significant changes since last release? https://screenshot.googleplex.com/7W5hL98RKSgmm7w
> > 
> > 
> > There are some critical vulnerabilities observed in the `endpoints-runtime-serverless` image tracked in b/373177830. https://screenshot.googleplex.com/8kF8c98DeWFdnkZ
> 
> I see. What is the diff between the head and last release? So does the new docker build env fix all those vulnerabilities?https://github.com/GoogleCloudPlatform/esp-v2/pull/930/files

As part of the release process, I verified that the latest [endpoints-runtime-serverless](https://pantheon.corp.google.com/artifacts/docker/cloudesf-testing/us/gcr.io/endpoints-runtime-serverless/sha256:66257969dc0118240bb4edf129782df14ed72eb423bf239af2ab36c5c04a97e0;tab=vulnerabilities?inv=1&invt=AbfVzw&project=cloudesf-testing) image has no critical vulnerability.

https://screenshot.googleplex.com/9tbeqewwuuxK4PQ

I believe the vulnerabilities were fixed for the ESPv2 dependencies in their latest versions that we are using.
